bootstrap/ansible/main.copyme

```diff
@@ -223,6 +231,10 @@ common_tasks: True
 # # TODO: For LRC, use the substring 'MyLRC'.
 # email_subject_prefix: '[MyBRC-User-Portal]'
 
+# # A list of admin email addresses to be notified about new requests and other
+# # events.
+# # TODO: Set these addresses to yours.
+# email_admin_list: ['you@email.com']
```

Comment:
Would an empty list default value work here? It would be one less thing to edit when re-filling up main.yml

Reply:
To add on to this, would placeholder values for emails, and default users and passwords set to "admin" and "root" or similar work for the dev section in `main.yml`? There probably shouldn't be a need to have strong passwords and emails sent out from dev. Then the workflow for testing main.yml changes could be optimized to literally just copy pasting `main.copyme` and uncommenting the dev section. Plus, if the dev section was left uncommented by default, no changes would have to be made at all.

Reply:
- Yes, it can be left empty because `settings_template.tmpl` has a default when it's empty. When `DEBUG` is `True`, email recipients get swapped out anyway.
- I'm okay with setting dummy values (emails, passwords, etc.) for dev--**do you mind making that change in a separate branch**? I generally don't fully copy `main.copyme` into `main.yml` after the first time; I just make the necessary changes on a case-by-case basis. But setting dummy defaults would make simply copying the file easier.
    - One potential edge case: we'd need to check if the playbook will update existing passwords (e.g., for Postgres) if they change.
- If the dev section were uncommented, we'd have to comment it out on staging and production. But, again, I don't re-copy the file after the first time, so this is less of an issue for me. I don't feel strongly either way about uncommenting it.